test/contexts_spec.lua

```diff
@@ -111,9 +168,9 @@ for _, lang in ipairs(langs_with_queries) do
         return
       end
 
-      setup(function()
+      lazy_setup(function()
```

Comment:
Would running busted with `--lazy` be an option?

Reply:
theoretically, but doesn't seem to work for me :/